keystone.js

```diff
@@ -17,10 +17,14 @@ keystone.initExpressApp();
 var keystoneServer = http.createServer(keystone.app);
 
 var shutdown = require('./lib/shutdown');
+
 keystoneServer = shutdown.addGracefulShutdownHook(keystoneServer, {
-  onSignal: shutdown.cleanupKeystone(keystone),
-  onShutdown: shutdown.logShutdownService('keystone'),
-});
+	healthChecks: {
+		'/healthcheck': require('./lib/healthcheck').createHealthCheckHandler(keystone),
+	},
+	onSignal: shutdown.cleanupKeystone(keystone),
+	onShutdown: shutdown.logShutdownService('keystone'),
+})
```

Comment:
Q: do we need to add 
`statusOk,                         // [optional = 200] status to be returned for successful healthchecks`
`statusError,                      // [optional = 503] status to be returned for unsuccessful healthchecks`

in the default options for `createTerminus` function ?

Reply:
statusOk is default to 200. statusError is default to 503.